asf_heat_pump_suitability/config/base.yaml

```diff
@@ -2,8 +2,8 @@ data_source:
   gb_ons_postcode_dir_url: "https://www.arcgis.com/sharing/rest/content/items/487a5ba62c8b4da08f01eb3c08e304f6/data" # Aug 2023 data
```

Comment:
This is a great idea. Thank you for taking the time to do this! We should definitely clean the data sources in here as it is getting messy and confusing to find things. I especially like that you came up with the ["EW"] and ["S"] keys!

I will suggest we create a separate issue and PR for this as this would require a lot of refactoring. Although it should be easy to do, we have to do it carefully to ensure that all the places where the `config` is called is updated with the appropriate keys.

We also will have to update the `config/README` `config key` column.

Reply:
I have opened a new issue #92 for this :) thank you!